dist/leaflet.css

```diff
@@ -273,6 +276,10 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	font: 0.75rem/1.5 "Helvetica Neue", Arial, Helvetica, sans-serif;
 	}
 
+.leaflet-rtl{
+	direction: rtl;
+	}

```

Comment:
Is this at all needed?

Reply:
Yes. If the user decides that leaflet has direction `rtl` but not the parent of the map container, then it would not work. (visible in the attribution control)